Source/JavaScriptCore/assembler/MacroAssemblerX86_64.h

```diff
@@ -2901,19 +2901,19 @@ class MacroAssemblerX86_64 : public MacroAssemblerX86Common {
         RELEASE_ASSERT(supportsAVX());
         ASSERT(scalarTypeIsFloatingPoint(simdInfo.lane));
         if (simdInfo.lane == SIMDLane::f32x4)
-            m_assembler.vminps_rrr(right, left, dest);
+            m_assembler.vminps_rrr(left, right, dest);
         else
-            m_assembler.vminpd_rrr(right, left, dest);
+            m_assembler.vminpd_rrr(left, right, dest);
```

Comment:
Does this mean that x64 and ARM64 have different semantics on Pmin?
If so, let's add comment describing why this order matters.

Reply:
Added a comment above in the latest push.